src/default-options.js

```diff
@@ -3,7 +3,8 @@ import path from 'path';
 
 export default function (log) {
   let defaultOptions = {
-    endOfLineChar: os.EOL
+    endOfLineChar: os.EOL,
+    fsPollInterval: 250
```

Comment:
My personal preference would be to have the watch poll slightly less frequently. The data isn't super time-sensitive (though it would be nice to have sooner to better plan a next move) so I'd prefer not to have the additional overhead.

Maybe something like?:
```suggestion
    fsPollInterval: 1250
```